README.md

```diff
@@ -10,9 +10,58 @@ This library requires a recent (at least 1.22) version of [deno](https://deno.la
 
 Any invocations of this library require the `--allow-read` and `--allow-net` [deno permissions](https://deno.land/manual/getting_started/permissions).
 
-## Script Overrides
+## Supported Scripts
+The hooks currently provided by this repo are `build`, `start`, `check-update`, `install-update`, `get-trigger`, and `get-manifest`.
+
+| Hook Name         | CLI Command            | Description                     |
+| ----------------- | ---------------------- | ------------------------------- |
+| `build`           | `slack deploy`         | Bundles any functions with Deno into an output directory that's compatible with the Run on Slack runtime. For more information, see the [deno-slack-builder](https://github.com/slackapi/deno-slack-builder) repository. |
+| `check-update`    | `slack upgrade`        | Checks the App's SDK dependencies to determine whether or not any of your libraries need to be updated. |
+| `get-manifest`    | `slack manifest`       | Converts a `manifest.json`, `manifest.js`, or `manifest.ts` file int o a valid manifest JSON payload. For more information, see the [deno-slack-builder](https://github.com/slackapi/deno-slack-builder) repository's `--manifest` arg. |
+| `get-hooks`       | N/A                    | Fetches the list of available hooks for the CLI from this repository. |
+| `get-trigger`     | `slack trigger create` | Converts a specified `json`, `js`, or `ts` file into a valid trigger JSON payload to be uploaded by the CLI to the `workflows.triggers.create` Slack API endpoint. |
+| `install-update`  | `slack upgrade`        | Prompts the user to automatically update any dependencies that need to be updated based on the result of the `check-update` hook. |
+| `start`           | `slack run`            | Creates a socket connection between the Slack CLI and a Slack workspace for local development that includes hot reloading. For more information, see the [deno-slack-runtime](https://github.com/slackapi/deno-slack-runtime) repository's details on `local-run`. |
```

Comment:
Hate to be pedantic, but this is not technically correct - the CLI manages the connection for deno apps. Here's my suggestion - feel free to ignore as well.

```suggestion
| `start`           | `slack run`            | While developing and locally running a deno-slack-based application, the CLI manages a socket connection with Slack's backend and delegates to this hook for invoking the correct application function for relevant events incoming via this connection. For more information, see the [deno-slack-runtime](https://github.com/slackapi/deno-slack-runtime) repository's details on `local-run`. |
```

Reply:
So the CLI would treat non-Deno apps differently if they implemented the same logic that's in our hook?

Reply:
Yes, the details for this hook are subtle but that's true. Bolt, for example, handles the websocket connection, so the CLI doesn't need to worry about it. Check out the CLI/SDK interface spec quip doc, under the 'list of hooks', you'll see two entries for the `start` hook. The difference is what the value of the config field in the `slack.json` is (`managed_connection: true` vs. `managed_connection: false`)

Reply:
You can also take a look and [see how the CLI acts differently based on the value of this `managed_connection` configuration](https://github.com/slackapi/slack-cli/blob/main/pkg/cloud/run.go#L134-L153).